spec/spec_helper.rb

```diff
@@ -25,6 +25,10 @@ module Methods
       def read_fixture(name)
         File.read(File.join(File.expand_path('..', __FILE__), 'fixtures', name))
       end
+
+      def read_json_fixture(name)
+        JSON.load(read_fixture(name))
```

Comment:
Guess I should have used ```JSON.parse``` instead.

Reply:
yep. Just spotted that. Commit amended and repushed.